python/pyarrow/parquet/core.py

```diff
@@ -2338,9 +2340,10 @@ def __init__(self, path_or_paths, filesystem=None, *, filters=None,
         if decryption_properties is not None:
             read_options.update(decryption_properties=decryption_properties)
 
-        # map filters to Expressions
-        self._filters = filters
-        self._filter_expression = filters and _filters_to_expression(filters)
+        # map filters to Expressions if not already
+        if not isinstance(filters, ds.Expression):
```

Comment:
The `_filters_to_expression` function has this check already as well, so this shouldn't be needed?

Reply:
If I understand correctly, `_filters_to_expression` doesn't accept `Expression` type. So here, if it's already an `Expression` then no processing is needed.